usr/lib/common/mech_ec.c

```diff
@@ -26,6 +26,57 @@
 #include "tok_specific.h"
 #include "ec_defs.h"
 
+#include "openssl/obj_mac.h"
+#include <openssl/ec.h>
+
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
check for < 1.0.1 is pkay for the brainpool defines, but for BN_bn2binpad it should be < 1.1.0 afaik.

Reply:
fixed